client/src/components/new-request/search.vue

```diff
@@ -1,32 +1,42 @@
 <template>
   <v-container fluid id="new-request-container" class="copy-normal">
     <v-row cols="12">
-      <v-col cols="6" class="font-weight-bold h6">I need a name to:</v-col>
+      <v-col cols="6" class="font-weight-bold h6"><span>I need a name to:</span></v-col>
       <v-col cols="6">
         <span id="nr-required-activator"
               class="link-std d-flex justify-end"
               @click="activateNRRModal()">Check to see if you need to file a a Name Request</span>
       </v-col>
-      <v-col cols="5">
+      <v-col cols="5 mt-n1">
         <!--request_action_cd-->
-        <v-select :error-messages="errors.includes('request_action_cd') ? 'Please select a type' : ''"
-                  :hide-details="!errors.includes('request_action_cd')"
-                  :items="requestActions"
-                  @change="clearErrors()"
-                  filled
-                  id="search-type-options-select"
-                  v-model="request_action_cd">
-          <template slot="item" slot-scope="data">
-            <v-tooltip :disabled="!data.item.blurbs" right transition="fade-transition">
-              <template v-slot:activator="scope">
-                <span v-on="scope.on" class="list-item">{{ data.item.text }}</span>
-              </template>
-              <span>{{ data.item.blurbs }}</span>
-            </v-tooltip>
+        <v-tooltip top
+                   id="search-type-options-select"
+                   content-class="top-tooltip"
+                   transition="fade-transition"
+                   :disabled="request_action_cd !== 'CNV'">
```

Comment:
Are these constants/enums that can be used here?

Reply:
I don't believe there is global enum for the `request_action_cd` values. Though, i think it's a good idea for the near future!

Reply:
Haha, OK!

Add to that an enum for NR state and another for NR actions.

Reply:
Would you mind create a td ticket for ^^ please ?

Reply:
Couldn't agree more. Will do!

Reply:
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5736